<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaterialForm</class>
 <widget class="QWidget" name="MaterialForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>246</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Material Form</string>
  </property>
  <layout class="QFormLayout" name="form_layout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
   </property>
   <property name="horizontalSpacing">
    <number>4</number>
   </property>
   <property name="verticalSpacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label_name">
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="name"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_unknown0">
     <property name="text">
      <string>Unknown 0:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="unknown0">
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_cull_mode">
     <property name="text">
      <string>Cull Mode:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="ComboBox" name="cull_mode"/>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QGroupBox" name="group_alpha_test">
     <property name="title">
      <string>Alpha Test</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QFormLayout" name="layout_alpha_test">
      <property name="horizontalSpacing">
       <number>4</number>
      </property>
      <property name="verticalSpacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_alpha_test_function0">
        <property name="text">
         <string>Function 0:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="ComboBox" name="alpha_test_function0"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_alpha_test_reference0">
        <property name="text">
         <string>Reference 0:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="alpha_test_reference0">
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_alpha_test_function1">
        <property name="text">
         <string>Function 1:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="ComboBox" name="alpha_test_function1"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_alpha_test_reference1">
        <property name="text">
         <string>Reference 1:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="alpha_test_reference1">
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_alpha_test_operator">
        <property name="text">
         <string>Operator:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="ComboBox" name="alpha_test_operator"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QGroupBox" name="group_depth_mode">
     <property name="title">
      <string>Depth Mode</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QFormLayout" name="group_depth_mode_layout">
      <property name="horizontalSpacing">
       <number>4</number>
      </property>
      <property name="verticalSpacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="3" column="0">
       <widget class="QLabel" name="label_depth_mode_function">
        <property name="text">
         <string>Function:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="ComboBox" name="depth_mode_function"/>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QCheckBox" name="depth_mode_test_early">
        <property name="text">
         <string>Test Early</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QCheckBox" name="depth_mode_enable">
        <property name="text">
         <string>Enable</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QCheckBox" name="depth_mode_update_enable">
        <property name="text">
         <string>Update Enable</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QGroupBox" name="group_blend_mode">
     <property name="title">
      <string>Blend Mode</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QFormLayout" name="group_blend__mode_layout">
      <property name="horizontalSpacing">
       <number>4</number>
      </property>
      <property name="verticalSpacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_blend_mode_function">
        <property name="text">
         <string>Function:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="ComboBox" name="blend_mode_function"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_blend_mode_source_factor">
        <property name="text">
         <string>Src. Factor:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="ComboBox" name="blend_mode_source_factor"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_blend_mode_destination_factor">
        <property name="text">
         <string>Dst. Factor:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="ComboBox" name="blend_mode_destination_factor"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_blend_mode_logical_operation">
        <property name="text">
         <string>Logic Op:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="ComboBox" name="blend_mode_logical_operation"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QCheckBox" name="dither">
     <property name="text">
      <string>Dither</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ComboBox</class>
   <extends>QComboBox</extends>
   <header>qt</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
